public/_redirects

```diff
@@ -179,5 +177,3 @@
 /studio /en/deprecated-software/#ethereum-studio 301!
 
 /*/studio /:splat/deprecated-software/#ethereum-studio 301!
-
-/en/stablecoins /stablecoins 301!
```

Comment:
Similar issue we have with `/staking` https://github.com/ethereum/ethereum-org-website/issues/12207. Those issues are now fixed with this PR.